### ISSUE TITLE
Need to resend initial email

### ISSUE THREAD

**Issue by Voyz** [formspree/formspree#105]
I've made a mistake of using formspree on a yet non-existent email address. Now that I've created it, formspree doesn't work since it's already sent the initial email with confirmation link to the address when it didn't yet existed.

Is there any way to resend that email or to fix it in another way?

**Comment by rohitdatta** [comment]
This was fixed in #66. Once you see the bounce error screen you'll need to fix the problem on your side and then you'll need to resend your form. You should now see a box to verify that you have fixed the error and to resend the confirmation email.

**Comment by Voyz** [comment]
Hey @rohitdatta, thanks for a quick reply.

Firstly, I never got a bounce error screen since I'm using AJAX version of formspree. The result I'm getting has a status of success and the data object contains a following string: _"confirmation email sent"_. Maybe it should return a different status on sequential attempts to sent confirmation email? At the moment the error is hidden from me quietly.

Going ahead with the non-AJAX form, I managed to get the bounce error screen, yet on hitting 'Resent confirmation email?' I got a following subpage:

> Verify the availability of your mailbox
> 
> We encountered an error when trying to deliver the confirmation message to \<my_email_here\> at the first time we tried. For spam reasons, we will not try again until we are sure the problem is fixed. Here's the reason:
> 
> 550 No Such User Here
> Please make sure this problem is not happening still, then come here and try to resend your confirmation message again.
> 

The email is up and running, I can send and receive emails. 

Is there anything else I need to do?

**Comment by rohitdatta** [comment]
Yep we should change the AJAX response, but we'll still need users to visit the Formspree site not using AJAX to confirm their inbox availability

As for your error, you'll need to POST again. That was originally designed to have users confirm their mailbox working, but I think that may be causing UX confusion so we can change it to have the confirmation checkbox show up on the error page too.

**Comment by rohitdatta** [comment]
If you have site-specific bugs, please write in to: [team@formspree.io](mailto:team@formspree.io) and we can help you there!

**Comment by Voyz** [comment]
@rohitdatta I've tried POSTing and requesting resend again and while this time it has not displayed the error and took me to a form of a confirmation page, there was still no email arriving to my address.

Also, I've used the team@formspree.io email to continue this conversation but neither this, nor the form available at the bottom of https://formspree.io/ seemed to have worked, as I haven't received any reply nor confirmation of a ticked being submitted.

**Comment by rohitdatta** [comment]
Sorry about that. We've been getting a lot of support requests lately, and it's caused quite a backlog. I'll try to take a look and see what I can do.
